packages/snowpack/src/commands/paint.ts

```diff
@@ -3,10 +3,26 @@ import {EventEmitter} from 'events';
 import * as colors from 'kleur/colors';
 import path from 'path';
 import readline from 'readline';
-import util from 'util';
-import {isYarn} from '../util';
 const cwd = process.cwd();
 
+export const paintEvent = {
+  BUILD_FILE: 'BUILD_FILE',
+  LOAD_ERROR: 'LOAD_ERROR',
+  CONSOLE_INFO: 'CONSOLE_INFO',
+  CONSOLE_WARN: 'CONSOLE_WARN',
+  CONSOLE_ERROR: 'CONSOLE_ERROR',
+  SERVER_RESPONSE: 'SERVER_RESPONSE',
+  SERVER_START: 'SERVER_START',
+  WORKER_COMPLETE: 'WORKER_COMPLETE',
+  WORKER_MSG: 'WORKER_MSG',
+  WORKER_RESET: 'WORKER_RESET',
+  WORKER_UPDATE: 'WORKER_UPDATE',
+};
+
+const MAX_CONSOLE_LENGTH = 500;
+const NO_COLOR_ENABLED = process.env.FORCE_COLOR === '0' || process.env.NO_COLOR;
```

Comment:
I don't know if this is necessary unless we want to support `NO_COLOR` which isn't supported by `kleur` yet. I opened a PR to address this, https://github.com/lukeed/kleur/pull/37

If we want to support `NO_COLOR` before this, we should be able to do `colors.$.enabled = !process.env.NO_COLOR && colors.$.enabled`. This should allow all `colors.*` calls to do nothing.

Reply:
Just kidding, Luke is on top of it!

https://github.com/lukeed/kleur/releases/tag/v4.1.0

Now we shouldn't need to add anything to support disabling colors.

Reply:
Nice!